django/views/decorators/common.py

```diff
@@ -9,8 +13,17 @@ def no_append_slash(view_func):
     # view_func.should_append_slash = False would also work, but decorators are
     # nicer if they don't have side effects, so return a new function.
     @wraps(view_func)
-    def wrapper_view(*args, **kwargs):
+    def _wrapper_view_sync(*args, **kwargs):
         return view_func(*args, **kwargs)
 
+    @wraps(view_func)
+    async def _wrapper_view_async(*args, **kwargs):
+        return await view_func(*args, **kwargs)
+
+    if asyncio.iscoroutinefunction(view_func):
+        wrapper_view = _wrapper_view_async
+    else:
+        wrapper_view = _wrapper_view_sync
+
```

Comment:
Here we're not actually consuming the response. 

I think it's worth looking at reduce the boilerplate in these cases, by simply marking the wrapper as a coroutine, [similar to in View.as_view()](https://github.com/django/django/blob/cbc0fb370583c6371eb2901f752ce286f008a6a4/django/views/generic/base.py#L118-L120). (When `wrapper_view` is called it will return the coroutine object, which will then be awaited by the calling code.)